### ISSUE TITLE
Array+Indices typealias

### ISSUE THREAD

**Issue by ochristensen** [nicklockwood/SwiftFormat#367]
```
extension MyCollection: RandomAccessCollection {
    typealias Indices = Array<MyCollectionItemType>.Indices
...
```
is being converted to 
```
extension MyCollection: RandomAccessCollection {
    typealias Indices = [MyCollectionItemType].Indices
...
```
which doesn't compile

**Comment by nicklockwood** [comment]
@ochristensen thanks for reporting this, I'll put up a fix as soon as possible. In the meantime, if there aren't too many cases then I suggest adding `// swiftformat:disable:next typeSugar` to the line above each.

**Comment by nicklockwood** [comment]
@ochristensen I've pushed a fix to the [develop branch](https://github.com/nicklockwood/SwiftFormat/tree/develop) if you'd like to try it.

**Comment by nicklockwood** [comment]
@ochristensen fixed in 0.39.3
